x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Verifier.java

```diff
@@ -288,6 +302,11 @@ private static boolean checkGroupByOrder(LogicalPlan p, Set<Failure> localFailur
         return true;
     }
 
+    private static boolean lala(Expression e, List<Expression> groupingAndMatchingAggregatesAliases) {
```

Comment:
?

Reply:
:( Tried to extract a method but not necessary, sorry!